style.css

```diff
@@ -0,0 +1,128 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+html,
+body {
+  font-size: 18px;
+  font-family: "Poppins", sans-serif;
+  width: 100%;
+  overflow-x: hidden;
+}
+
+.navbar {
+  background-color: white;
+}
+
+.navbar-content {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-left: 40px;
+  margin-right: 40px;
+  margin-top: 40px;
+  border: none;
+}
+
+.navbar-content img {
+  width: 20px;
+  height: 13px;
+}
+
+.logo {
+  color: #6070ff;
+  font-weight: 700;
+  text-decoration: none;
+}
+
+.nav_link ul {
+  display: none;
+}
+
+.section-1 {
+  height: 720px;
+  background-color: #f7f7f9;
+  border-style: none;
+}
+
+.section-1-container {
+  background-image: url("images/header-shapes\ mobile.png");
+  background-repeat: no-repeat;
+  background-position: center;
+  border-bottom-left-radius: 100px;
+  background-color: white;
+  padding: 160px 25px;
+  width: 100%;
+  height: 100%;
+}
+
+.heading {
+  font-size: 2.5em;
+  font-weight: 700;
+  line-height: 52px;
+}
+
+.leading {
+  font-size: 18px;
+  text-align: start;
+  margin: 20px 0;
+}
```

Comment:
- [OPTIONAL] Kindly make sure your main introduction paragraph has the same color as the one on the provided Figma design. The used color is `#344563`, add it to your heading it will be identical and match the provided Figma design.

   - ### YOUR DESIGN
    
   ![image](https://user-images.githubusercontent.com/87197412/234824280-141a8157-ceca-456a-8601-9aca7eb5eee8.png)

   - ### FIGMA DESIGN
   
  ![image](https://user-images.githubusercontent.com/87197412/234824518-3449e879-9e9f-4d47-a6e6-20ec63c9cf7c.png)